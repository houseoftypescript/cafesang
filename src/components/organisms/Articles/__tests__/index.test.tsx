import { render } from '@testing-library/react';
import { Articles } from '../index';

describe('Button', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Articles loading={true} error={null} articles={[]} />
    );
    expect(container).toMatchSnapshot();
  });
});
