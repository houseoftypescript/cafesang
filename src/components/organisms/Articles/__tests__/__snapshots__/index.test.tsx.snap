// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button to match snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      class="py-16 text-center"
    >
      <p
        class="uppercase text-xl"
      >
        Loading
      </p>
    </div>
  </div>
</div>
`;
